contrib/mod_sftp/mod_sftp.h.in

```diff
@@ -69,7 +79,7 @@
 
 #if defined(HAVE_LIBRESSL) && \
   LIBRESSL_VERSION_NUMBER < 0x3010000fL
-# undef HAVE_BROKEN_CHACHA20
+# define HAVE_BROKEN_CHACHA20	1
```

Comment:
The _intent_ here was to declare/define that older LibreSSL versions had broken ChaCha20 support; that intent was not actually implemented properly in the past.  Oops.